package.json

```diff
@@ -5,19 +5,33 @@
   "scripts": {
     "build": "tsc",
     "test": "mocha",
-    "lint": "tslint '*(test|src)/**/*.ts'"
+    "lint": "tslint '*(test|src)/**/*.ts'",
+    "start": "tsc && node dist/index.js"
```

Comment:
Why do you need to run the index?

Reply:
I was using it to code a test version of the class TestxServer before moving it to testx-server folder, but I'll remove